tests/test_health_check.py

```
@@ -0,0 +1,223 @@
+
+    def test_local_in_memory(self):
+        client = QdrantClient(":memory:")
+        assert client.health_check() is True
+
+    def test_local_after_close(self):
+        client = QdrantClient(":memory:")
+        client.close()
+        assert client.health_check() is False
+
+    def test_local_path(self):
+        client = QdrantClient(path="/tmp/qdrant_health_test")
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/qdrant_health_test"

(S108)